src/Core/Prefab/Globe/GlobeLayer.js

```diff
@@ -0,0 +1,72 @@
+import * as THREE from 'three';
+
+import TiledGeometryLayer from '../../../Layer/TiledGeometryLayer';
+
+import { processTiledGeometryNode } from '../../../Process/TiledNodeProcessing';
+import { globeCulling, preGlobeUpdate, globeSubdivisionControl, globeSchemeTileWMTS, globeSchemeTile1 } from '../../../Process/GlobeTileProcessing';
+import BuilderEllipsoidTile from './BuilderEllipsoidTile';
+
+class GlobeLayer extends TiledGeometryLayer {
+    /**
+     * A {@link TiledGeometryLayer} to use with a {@link GlobeView}. It has
+     * specific method for updating and subdivising its grid.
+     *
+     * @constructor
+     * @extends TiledGeometryLayer
+     *
+     * @param {string} id - The id of the layer, that should be unique. It is
+     * not mandatory, but an error will be emitted if this layer is added a
+     * {@link View} that already has a layer going by that id.
+     * @param {THREE.Object3d} [object3d=THREE.Group] - The object3d used to
+     * contain the geometry of the TiledGeometryLayer. It is usually a
+     * <code>THREE.Group</code>, but it can be anything inheriting from a
+     * <code>THREE.Object3d</code>.
+     * @param {Object} [config] - Optional configuration, all elements in it
+     * will be merged as is in the layer. For example, if the configuration
+     * contains three elements <code>name, protocol, extent</code>, these
+     * elements will be available using <code>layer.name</code> or something
+     * else depending on the property name.
+     * @param {number} [config.maxSubdivisionLevel=18] - Maximum subdivision
+     * level for this tiled layer.
+     * @param {number} [config.sseSubdivisionThreshold=1] - Threshold level for
+     * the SSE.
+     * @param {number} [config.maxDeltaElevationLevel=4] - Maximum delta between
+     * two elevations tile.
+     *
+     * @throws {Error} <code>object3d</code> must be a valid
+     * <code>THREE.Object3d</code>.
+     */
+    constructor(id, object3d, config = {}) {
+        config.defaultPickingRadius = 5;
+
+        super(id, object3d || new THREE.Group(), config);
+
+        // Configure tiles
+        this.schemeTile = globeSchemeTileWMTS(globeSchemeTile1);
+        this.extent = this.schemeTile[0].clone();
+        for (let i = 1; i < this.schemeTile.length; i++) {
+            this.extent.union(this.schemeTile[i]);
+        }
+
+        function subdivision(context, layer, node) {
+            if (TiledGeometryLayer.hasEnoughTexturesToSubdivide(context, node)) {
+                return globeSubdivisionControl(2,
```

Comment:
* Could you standardize the signature of  `globeSubdivisionControl`, `panoramaSubdivisionControl` and `planarSubdivisionControl`, because these functions are pretty much the same

* Could you move `subdivision` and `this.update` declaration to `TiledGeometryLayer`

Reply:
- pretty much yes, but the panorama one is kinda annoying... I'll see what I can do here
- I don't agree with this, I'd prefer to transfer the called methods (from `GlobeTiledProcessing` for example) in layers in the PR following this one.